linkerd/app/inbound/src/port_policies.rs

```diff
@@ -76,52 +81,120 @@ impl From<DefaultPolicy> for PortPolicies {
     }
 }
 
-impl From<AllowPolicy> for PortPolicies {
-    fn from(default: AllowPolicy) -> Self {
+impl From<ServerPolicy> for PortPolicies {
+    fn from(default: ServerPolicy) -> Self {
         DefaultPolicy::from(default).into()
     }
 }
 
-// === impl DefaultPolicy ===
+impl PortPolicies {
+    pub(crate) fn check_allowed(
```

Comment:
nit/tioli: it might be nice if there was some kind of comment someplace explaining the relationship between `check_allowed` and `check_authorized` here?